src/js/stores/policy.js

```diff
@@ -96,8 +104,13 @@ define(function (require, exports, module) {
 
         /**
          * Emits a change event - used by debug policy overlay
+         *
+         * @private
+         * @param {{kind: string}} payload
          */
-        _emitChange: function () {
+        _handlePoliciesInstalled: function (payload) {
+            delete this._dirtyPolicies[payload.kind];
```

Comment:
This event means we just sync'd policies, which means this policy kind is no longer dirty.